src/common/redis_module/ray_redis_module.cc

```diff
@@ -431,6 +470,68 @@ int TableLookup_RedisCommand(RedisModuleCtx *ctx,
   return REDISMODULE_OK;
 }
 
+bool is_nil(const std::string &data) {
+  CHECK(data.size() == 20);
+  const uint8_t *d = reinterpret_cast<const uint8_t *>(data.data());
+  for (int i = 0; i < 20; ++i) {
+    if (d[i] != 255) {
+      return false;
+    }
+  }
+  return true;
+}
+
+// This is a temporary redis command that will be removed once
+// the GCS uses https://github.com/pcmoritz/credis.
+// Be careful, this only supports Task Table payloads.
+int TableTestAndUpdate_RedisCommand(RedisModuleCtx *ctx,
```

Comment:
Is the plan for this to become generic and handle other tables as well?

Reply:
I don't see a way to do that but it would be cool. At the moment the task table is the only table that has test and update.